doc/adr/0006-centralised-openapi-spec-storage.md

```
@@ -0,0 +1,57 @@
+service module, making it hard for the app and other consumers to discover and
+use them, and creating duplication when both a server and a client need to
+generate code from the same spec.
+
+## Decision
+
+All microservice APIs must be documented using the OpenAPI specification format.
+
+All OpenAPI specs are stored in a central `api-specs/` directory at the
+repository root. Each service has its own subdirectory:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)